gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/AccountController.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 Apple Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.controllers
+
+import com.netflix.spinnaker.gate.services.internal.ClouddriverService
+import io.swagger.annotations.Api
+import io.swagger.annotations.ApiOperation
+import io.swagger.annotations.ApiParam
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.access.prepost.PostFilter
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@Api
+class AccountController {
```

Comment:
Looks like there's some duplication with functionality in the [CredentialsController](https://github.com/spinnaker/gate/blob/ecc999c3d62a48d2c297497e94829c85dfa2fa72/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy).  Could these methods go there instead?

Reply:
I think they might be able to be merged together, yeah.

Reply:
Looking more closely, this seems fairly awkward to merge in here. Any suggested ways of organizing the REST API within `/credentials` as the root directory?

Reply:
Planning to combine these into `/credentials` by omitting the duplicate endpoints (plus, why bother returned the decrypted secret via the endpoint in the first place?)

Reply:
I've combined the new controller into the credentials one, but now I'm getting a Spring error about there being two conflicting bean methods for `GET /credentials`.